cassandra-analytics-core/src/main/java/org/apache/cassandra/clients/Sidecar.java

```diff
@@ -129,6 +135,9 @@ public static SidecarClient from(SidecarInstancesProvider sidecarInstancesProvid
                                             .ssl(conf.hasKeystoreAndKeystorePassword())
                                             .build();
 
+        StartupValidator.instance().register(new KeystoreValidation(secretsProvider));
```

Comment:
secretsProvider is only available in the SBR context. For SBW, we'll need to check whether keystore/password and/or truststore/password are provided, and validate

Reply:
Fixed this place using a new validator.